pom.xml

```diff
@@ -312,6 +312,12 @@ Import-Package: \\
           <version>2.8.2</version>
         </plugin>
 
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-enforcer-plugin</artifactId>
+          <version>3.0.0-M2</version>
```

Comment:
Is this a release?

Reply:
It is on Maven Central and there you can find releases only:
https://search.maven.org/artifact/org.apache.maven.plugins/maven-enforcer-plugin/3.0.0-M2/maven-plugin